src/main/java/com/redhat/rcm/offliner/Options.java

```diff
@@ -27,6 +27,8 @@
 public class Options
 {
 
+    public static final String DEFAULT_TYPE_MAPPING = "test-jar=jar:tests;ejb=jar;ejb-client=jar:client;maven-plugin=jar;java-source=jar:sources;javadoc=jar:javadoc";
+
```

Comment:
would this be better as a properties file contained in the jar, with a reference to it here?